app/facades/trip_facade.rb

```diff
@@ -7,67 +7,40 @@ def initialize(trip_params)
   end
 
   def response
-    # Origin / destination names from Google Maps
-    start_address = _google_maps_service.start_address
-    end_address = _google_maps_service.end_address
-
-    # Get long/lat of origin and destination
-    origin_lat = _google_maps_service.origin_lat
-    origin_lng = _google_maps_service.origin_lng
-    destination_lat = _google_maps_service.destination_lat
-    destination_lng = _google_maps_service.destination_lng
-
-    # Get miles and time between origin and destination
-    distance_text = _google_maps_service.distance_text
-    duration_hours = _google_maps_service.duration_hours
-
-    # Get population or origin and destination
-    origin_population = Poi.population_at_location(origin_lat, origin_lng)
-    destination_population = Poi.population_at_location(destination_lat, destination_lng)
-
-    # Get weather of origin currently
-    origin_weather = ForecastSerializer.new(_weather_service_origin.json).currently
-
-    # Get weather of destination at X hours from now
-    destination_weather = ForecastSerializer.new(_weather_service_destination.json).hourly_in(duration_hours.round)
+    origin = _poi_service.places[0]
+    origin_weather = ForecastSerializer.new(_weather_service.get(origin.center.lat, origin.center.lng)).currently
```

Comment:
This Weather serialization/service is confusing and mostly my fault. I'll make a card to refactor this.